### ISSUE TITLE
Not working for IOS

### ISSUE THREAD

**Issue by sravanpronteff** [react-native-picker/picker#341]
<Picker
  selectedValue={selectedLanguage}
  onValueChange={(itemValue, itemIndex) =>
    setSelectedLanguage(itemValue)
  }>
  <Picker.Item label="Java" value="java" />
  <Picker.Item label="JavaScript" value="js" />
</Picker>

not working on IOS from expo

**Comment by florianf22** [comment]
experiencing the same issue. can't get it working no matter what

**Comment by EnzoOctupus** [comment]
Same Problem here

**Comment by davormaric53** [comment]
Same problem here. I have discovered that the style property **alignItems: 'center'** is causing some problems. With it only the background colour is shown. Without it the dialogue list is shown even if the mode is set to the dropdown. On Android, it looks fine

**Comment by tendaimaswaya** [comment]
Facing the same issue. Has anyone found a solution to this yet?

**Comment by bll-poojapiprotar** [comment]
'React/UIView+React.h' file not found.

**Comment by PoojaPiprotar** [comment]
Facing issue in IOS, in android it's working fine. 
'React/UIView+React.h' file not found

**Comment by CodingLime** [comment]
Also facing the same issue in iOS. On Android it's working normally.
Any tips/workarounds for now ?

**Comment by yesIamFaded** [comment]
Is there some more information? I removed some if the styles but it still does not work it wont even show the pickers on IOS, Android is perfect.

**Comment by pedro-ernesto** [comment]
Hey guys!

Had some style issues, this made it work for me (using styled-components):

Removed **flex-direction**, **justify-content** and **align-items**

**Comment by bytesagar** [comment]
Why isn't someone posting some real solution for this, this package doesn't work with IOS why is that. I tried everything.

**Comment by noemipisano96** [comment]
I ran in a similar issue.
My project is managed with expo 44. 
While the app is running in Expo Client (locally or published in channels) everything works fine, but when I build an ipa or apk and install on device my app crashes.

Any solution?

**Comment by PaxLyj** [comment]
It seems like Picker on iOS need to be specified `width`. I just solved same issue just give `width: '100%'`.

**Comment by bytesagar** [comment]
@noemipisano96 yes installing react-native-picker/picker solved the issue for me.

**Comment by markedwards** [comment]
I also found that it was necessary to set `itemStyle`, because the default text color was the same as my background, so the text was invisible.

**Comment by jarig** [comment]
It actually exactly the same issue on Android too, if `align-items: center` is set on any of parent views (even transitively) - it doesn't show anything then.
